sdk/storage/storage-blob/samples/javascript/basic.js

```diff
@@ -19,24 +13,31 @@ async function main() {
   // SharedKeyCredential is only avaiable in Node.js runtime, not in browsers
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
-  // Use RawTokenCredential with OAuth token.  You can find more
-  // TokenCredential implementations in the @azure/identity library
-  // to use client secrets, certificates, or managed identities for
-  // authentication.
-  const tokenCredential = new RawTokenCredential("token");
-  tokenCredential.token = "renewedToken"; // Renew the token by updating token field of token credential
+  // DefaultAzureCredential will first look for Azure Active Directory (AAD)
```

Comment:
Do we need the separate azureAd.js/ts sample if we also have this inside of the basic sample?  If you feel that we don't, feel free to delete it

Reply:
Just following a convention where all the credential methods are mentioned in the basic.ts(and basic.js sample) along with the individual samples for the each credential method.